pkg/apis/kops/validation/validation.go

```diff
@@ -1397,6 +1406,23 @@ func validateNetworkingGCP(cluster *kops.Cluster, v *kops.GCPNetworkingSpec, fld
 	return allErrs
 }
 
+func validateNetworkingKindnet(cluster *kops.Cluster, v *kops.KindnetNetworkingSpec, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if v.Masquerade != nil && v.Masquerade.Enabled != nil && *v.Masquerade.Enabled {
```

Comment:
Nit: We have ValueOf, `v.Masquerade.Enabled != nil && *v.Masquerade.Enabled` === `fi.ValueOf(v.MasqueradeEnabled)`